go/cmd/vtctldclient/command/transactions.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2024 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"vitess.io/vitess/go/cmd/vtctldclient/cli"
+	querypb "vitess.io/vitess/go/vt/proto/query"
+	vtctldatapb "vitess.io/vitess/go/vt/proto/vtctldata"
+)
+
+var (
+	DistributedTransaction = &cobra.Command{
+		Use:   "DistributedTransaction <cmd>",
+		Short: "Perform commands on distributed transaction",
+		Args:  cobra.MinimumNArgs(2),
+
+		DisableFlagsInUseLine: true,
+	}
+
+	// GetUnresolvedTransactions makes an GetUnresolvedTransactions gRPC call to a vtctld.
+	GetUnresolvedTransactions = &cobra.Command{
+		Use:     "list <keyspace>",
+		Short:   "Retrieves unresolved transactions for the given keyspace.",
+		Aliases: []string{"List"},
+		Args:    cobra.ExactArgs(1),
+		RunE:    commandGetUnresolvedTransactions,
+
+		DisableFlagsInUseLine: true,
+	}
+
+	// ConcludeTransaction makes a ConcludeTransaction gRPC call to a vtctld.
+	ConcludeTransaction = &cobra.Command{
+		Use:     "conclude <dtid> [<keyspace/shard> ...]",
+		Short:   "Concludes the unresolved transaction by rolling back the prepared transaction on all participating shards and removing the transaction metadata record.",
```

Comment:
```suggestion
		Short:   "Concludes the unresolved transaction by rolling back prepared transactions on all participating shards and removing the transaction metadata record.",
```

Reply:
every shard will have at most 1 prepared transaction.

Reply:
I believe it is how we see this distributed transaction. 
one transaction that spans multiple shards
or multiple independent transactions on the shards.

For user, it is one transaction.
Internally it is multiple transactions.

Reply:
if you want to use singular, you can use `each`
```
Concludes the unresolved transaction by rolling back the prepared transaction on each participating shard and removing the transaction metadata record
```
OR
```
Concludes the unresolved transaction by rolling back prepared transactions on all participating shards and removing the transaction metadata record
```
are both grammatically correct. Using the singular with "all" is not.

Reply:
I will take the first one

Reply:
@deepthi could you also approve the PR if everything else looks good?